interface/usergroup/facilities_add.php

```diff
@@ -142,15 +140,6 @@ function toggle( target, div ) {
     $("#cancel").click(function() {
           dlgclose();
      });
-
-    /**
-     * add required/star sign to required form fields
-     */
-    for (var prop in collectvalidation) {
```

Comment:
Was this feature removed from this script?

Reply:
We need to find a different way to say there is required fields cause this wasn't working.